v2/ci.py

```diff
@@ -13,6 +13,9 @@
 p.add("--test-dry-run", default="False")
 p.add("--test-focus-regex", default="\\[Conformance\\]|\\[NodeConformance\\]|\\[sig-windows\\]")
 p.add("--test-skip-regex", default="\\[LinuxOnly\\]")
+p.add("--test-flake-attempts", default="1")
+p.add("--test-num-nodes", default="2")
```

Comment:
No need to add test-num-nodes parameter. The value for test num nodes is the number of windows nodes in the cluster.